lib/common/models/wp_user/existable.rb

```diff
@@ -76,7 +76,10 @@ def self.display_name_from_body(body)
         # replace UTF chars like  &#187; with dummy character
         title_tag.gsub!(/&#(\d+);/, '|')
 
-        name = title_tag[%r{([^|«»]+) }, 1]
+        name = title_tag[%r{.+» (.+)}, 1]

```

Comment:
I like this method better than the current checks of the display name (ours and yours) even if it's still theme dependent, but negate the regexp madness (sometimes the display name appears first, e.g: 'admin « Some Title')